libraries/AP_UAVCAN/AP_UAVCAN_IfaceMgr.h

```diff
@@ -17,7 +17,7 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#if HAL_MAX_CAN_PROTOCOL_DRIVERS
+#if HAL_ENABLE_LIBUAVCAN_DRIVERS
```

Comment:
So this should really be `HAL_LIBUAVCAN_DRIVERS_ENABLED`, but it is pre-existing.